backend/load_dv1_places_problems.py

```diff
@@ -0,0 +1,273 @@
+from uuid import uuid4
```

Comment:
I: I've been trying to keep the extract/transform logic separate from the app in [pipelines](https://github.com/districtr/districtr-v2/tree/main/pipelines). So you can prepare a dataset, dump it to bucket storage, and then separately load it to the app. I'd recommend following that pattern so that
- The app isn't responsible for brittle pipeline logic
- The pipeline logic is more easily adjustable / testable if it's not coupled to postgres
- We have a single artifact in S3 that's easy to debug outside of the postgres database
- it's easier to understand what's going on since we have 1 artifact -> loaded into 1 table rather than hundreds/thousands of queries. This will also greatly improve load perf and not burden the prod DB when we run this